### ISSUE TITLE
Generate and implement new ideas for tags 

### ISSUE THREAD

**Issue by pet3r-pan** [ethereum-magicians/scrolls#37]
On the discourse forum, we are able to have tags as shown below. We can create any tag and give people tags on certain occasions or conditions etc. (eg. donated to ethmagicians = doner, or active member = no-lifer)

![image](https://user-images.githubusercontent.com/25356454/49559801-1d64ef00-f964-11e8-8bd3-91fea99dca60.png)

These tags can be made from text along with any icons from the font-awesome set.

Would love for this thread to be a place of grace peace and joyous come together for us to brain storm some ideas for new and spicy tags for people.

**Comment by jamesyoung** [comment]
Project/group affiliations? Possible to put in links to Twitter/Peepeth profile?

**Comment by bmann** [comment]
@pet3r-pan just clarifying that those "tags" are group affiliations.

We can create Discourse groups at will, and make them public / private or something in between. The public ones can be ones that users can join.

@MadeofTin wrote this up before, see https://github.com/ethereum-magicians/scrolls/issues/14
 
I would like to see this turned on. Probably this is actually a dupe of #14 -- what do you think Peter?

**Comment by pet3r-pan** [comment]
@bmann Yeah I think that these tag are for group affiliations (categorised rings) but also potentially for any other purpose eg. for MVP volunteers of councils (they get a cool unique tag such as Prague '18).

I just dug through metacartel's admin panel and yes, you can create groups where members automatically get the option using a new tag (or title) next to their username.

**Comment by bmann** [comment]
Using groups for Rings means mailing list mode for ring owners which is a little more effective than hoping people see category posts. 

And, open groups for Rings means people can self select what they want to show they are involved with. 

It also would mean we could identify people who are actually organizers of a group / Ring — which isn’t captured anywhere. 

Groups don’t really do categorization unless you send group PMs. I agree that categories as they are is fine.

**Comment by pet3r-pan** [comment]
For ring organisers, we can manually give them a badge or title.
